### PR TITLE
Add skizdigidy's Phoenix file

### LINKED ISSUE #9: skizdigidy's Workflow for adding Phoenix to the Repository
## Phoenix, AZ
- [x] Create an Issue
- [x] Create a Branch
- [x] Make Changes
- [x] Open a PR
- [ ] Collaborate/Have a discussion
- [ ] Get 👍  to 🚢
- [ ] Merge

### PR DESCRIPTION
**Pull request by skizdigidy**
closes #9

### CODE REVIEW

File: skizdigidy-phoenix.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+- Oreganos
+- Sun Asia
+
+### Places to Go
+- Camelback Mountain
+
+### Current Temperature
+- 76 degrees
```
Comment (skizdigidy) [inline skizdigidy-phoenix.md]: perfecto❕

File: skizdigidy-phoenix.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## Phoenix, AZ
 
 ### Restaurants
-- Oregonos
+- Oreganos
```
Comment (tlcoder) [inline skizdigidy-phoenix.md:4]: Close :pizza:

File: skizdigidy-phoenix.md (hunk 1 above)
Comment (BugmanMaster) [inline skizdigidy-phoenix.md:12]: About the same here.  Cool for Phoenix, no?

Comment (skizdigidy) [reply skizdigidy-phoenix.md:12]: Just starting to cool down.  Been a warm summer.

Comment (tlcoder) [reply skizdigidy-phoenix.md:12]: Do you like stuff?

File: skizdigidy-phoenix.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Phoenix, AZ
```
Comment (rice-ball-rolling) [inline skizdigidy-phoenix.md:1]: didn't we have cox outage this morning?

**Review by tlcoder — APPROVED** [review]

**Pull request merged**